### PR TITLE
update CODEOWNERS with OWNERS.md reference

### PR DESCRIPTION
**Pull request by displague**
add `@tinkerbell/site-eng` to https://github.com/tinkerbell/hegel/pull/29 and link to the OWNERS.md file that will be introduced by https://github.com/tinkerbell/.github/pull/4

### CODE REVIEW

**Comment by gianarb** [comment]
I think we should not do it
https://github.com/tinkerbell/hegel/pull/29#issuecomment-682399413

**Comment by displague** [comment]
After the discussion in the community meeting, @gianarb, Is this acceptable?

* tinkerbell/.github/OWNERS.md is a description of responsibilities (with no usernames)
* hegel/CODEOWNERS is a list of users and groups (with a link and comment to read ^ for more information)

**Comment by displague** [comment]
@gianarb would you rather see the OWNERS.md file in https://github.com/tinkerbell/org ?

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# These owners will be the default owners for everything in the repo. They
-# will be requested for review when someone opens a pull request.
-# We call those people maintainers as well
+# See [OWNERS.md](https://github.com/tinkerbell/.github/blob/master/OWNERS.md)
+# for clarity on the responsibilities of the CODEOWNERS.
+#
+# This "*" assignment defines the users that will be notified on all pull
+# requests.
 
+*   @tinkerbell/site-eng
```
Comment (gianarb) [inline CODEOWNERS:7]: Is this right?! previously it was only `site-eng` if I am not wrong

**Pull request merged**